crates/telio-traversal/src/connectivity_check/cross_ping_check.rs

```diff
@@ -63,6 +65,8 @@ mockall::mock! {
 
     #[async_trait]
     impl CrossPingCheckTrait for CrossPingCheckTrait {
+        async fn pause(&self, key: &PublicKey) -> Result<(), Error>;
+        async fn unpause(&self, key: &PublicKey) -> Result<(), Error>;
```

Comment:
`PublicKey` is `Copy` so maybe it makes sense to pass it by value?

Reply:
Sure makes sense. Fixed it :)